.github/workflows/test.yml

```diff
@@ -0,0 +1,41 @@
+name: build
+
+on: [ push, pull_request ]
+
+jobs:
+  run:
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        operating-system: [ ubuntu-latest ]
+        php-versions: [ '7.3', '7.4', '8.0' ]
+    name: PHP ${{ matrix.php-versions }} Test on ${{ matrix.operating-system }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
```

Comment:
Was there a reason you mentioned 'v1' and not 'master'?

Reply:
The `master` branch means it's not stable and that's the reason why I use the `v1` branch.

Reply:
I see. Thanks for the clarification 👌